### DISCUSSION TITLE
如何将一个引擎中的Local<String>转移到另一个引擎中

### DISCUSSION THREAD

**Discussion by yqs112358** [Tencent/ScriptX#29] [Q&A] [answered]
实现中，有实现跨引擎调用函数 / 传递数据的需要
请问有无直接的办法可以直接将一个引擎中的变量转移到另一个引擎当中？
还是说只能先转换到STL变量再到另一个引擎中重新使用工厂函数构建
@LanderlYoung

**Comment by LanderlYoung** [comment, marked as answer]
No. A Local<X> is a reference belong to a single engine.

get a std::string from one reference, and create another reference in the second engine.

**Reply by yqs112358** [reply]
Okok , Thanks a lot !
